schemas/mw-article-rename.js

```diff
@@ -0,0 +1,63 @@
+{
+    "title": "MediaWiki Article Rename",
+    "description": "Represents a MW Article rename event",
+    "type": "object",
+    "properties": {
+        // global event fields
+        "uri": {
+            "type": "string",
+            "format": "uri",
+            "description": "the unique URI identifying the event"
+        },
+        "request_id": {
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$",
+            "description": "the unique UUID v1 ID of the event derived from the X-Request-Id header"
+        },
+        "event_id": {
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$",
+            "description": "the unique ID of this event; added by the event bus"
+        },
+        "dt": {
+            "type": "string",
+            "format": "date-time",
+            "description": "the time stamp of the event, in ISO8601 format"
+        },
+        "domain": {
+            "type": "string",
+            "description": "the domain the event pertains to"
+        },
+        // event-specific fields
+        "title": {
```

Comment:
Suggest new_title to contrast with old_title (and correspond with comments around new/old pageIDs below)